flutter/alds/lib/settingspage.dart

```diff
@@ -129,7 +130,33 @@ class _AldsSettingsWidgetState extends State<AldsSettingsWidget> {
             }
         }
       ),
-    );
+      bottomSheet: Container(
```

Comment:
If possible, could you add more padding so the button appears a little further down the page? It is really close to the other setting features.